modules/AOW_Actions/actions/actionCreateRecord.php

```diff
@@ -170,12 +170,14 @@ function set_record(SugarBean $record, SugarBean $bean, $params = array(), $in_s
                             case 'int':
                                 $value = format_number($bean->$fieldName);
                                 break;
-                            case 'relate':
-                                if(isset($data['id_name'])) {
-                                    $idName = $data['id_name'];
+			    case 'relate':
+			        if(isset($data['id_name']) && $record_vardefs[$field]['type'] == 'relate'){
```

Comment:
You can safely use ===. Please can you change the == to ===?

Reply:
So sorry for late response. I have changed code as per you requested.